tests/tabnote_tests.ts

```diff
@@ -178,7 +178,7 @@ function drawStemsUp(options: TestOptions, contextBuilder: ContextBuilder): void
   stave.setContext(ctx);
   stave.draw();
 
-  const specs: TabNoteStruct[] = [
+  const specs: Partial<TabNoteStruct>[] = [
```

Comment:
If we consider that the `xxxx_tests.ts` files are examples of code that a developer using the VexFlow library would write, I would say that seeing all these instances of `Partial<TabNoteStruct>` are evidence that maybe we should avoid Partial, and instead just mark all fields as optional with ?.

@0xfe what do you think? Is it annoying for a dev to have to type out `Partial<TabNoteStruct>`? It would look slightly cleaner if the developer could just use `TabNoteStruct` directly.

Reply:
We have to use Partial due to the fact that we decided to use Partial definitions in the tests. If we defined the complete structure in the test Partial would not be required.